### ISSUE TITLE
Why is BootstrapVue still using "^4.0.0-beta.2", and not the Current v4.1.3?

### ISSUE THREAD

**Issue by BenHayat** [bootstrap-vue/bootstrap-vue#2097]
Project built shows dependencies on the 4 beta of Bootstrap.

**Comment by tmorehouse** [comment]
The current `master` is still at 4.0.0, while the new `dev` branch (where changes are being made) uses `^4.1.1`

See https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/package.json

**Comment by BenHayat** [comment]
> 
> 
> The current `master` is still at 4.0.0, while the new `dev` branch (where changes are being made) uses `^4.1.1`
> 
> See https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/package.json

But the new project I just created shows the following.

"dependencies": {
"vue": "^2.5.2",
"vue-router": "^3.0.1",
"bootstrap": "^4.0.0-beta.2",
"bootstrap-vue": "latest",
"popper.js": "^1.12.9"

**Comment by tmorehouse** [comment]
Which version of Bootstrap-Vue are you using? The version of Bootstrap V4 required will vary on which version of Bootstarp-Vue you are using.

**Comment by BenHayat** [comment]
> 
> 
> Which version of Bootstrap-Vue are you using? The version of Bootstrap V4 required will vary on which version of Bootstarp-Vue you are using.

@tmorehouse ;

Well, today is my first day of looking at bootstrap-Vue.
I went to https://bootstrap-vue.js.org/docs and in the section of Vue-Cli
I had previously installed Vue-cli 3. Now I entered
vue init bootstrap-vue/webpack my-project to create a full template. After it created that, I looked in the Package.json and noticed it's using the old BS beta 4.

I'm suspecting the "vue init bootstrap-vue/webpack" code is not up to date.
my Vue version is 3.0.3
